shiny/playwright/controller/_accordion.py

```diff
@@ -0,0 +1,321 @@
+from playwright.sync_api import Locator, Page
```

Comment:
Please add this import to the top of each of the python files. This will help with all of the python 3.9 and below lint errors

```suggestion
from __future__ import annotations

from playwright.sync_api import Locator, Page
```